app/assets/javascripts/admin/addon/components/ace-editor.js

```diff
@@ -95,67 +96,78 @@ export default class AceEditor extends Component {
 
   didInsertElement() {
     super.didInsertElement(...arguments);
+
     loadScript("/javascripts/ace/ace.js").then(() => {
-      window.ace.require(["ace/ace"], (loadedAce) => {
-        loadedAce.config.set("loadWorkerFromBlob", false);
-        loadedAce.config.set("workerPath", getURL("/javascripts/ace")); // Do not use CDN for workers
+      loadScript(`/javascripts/ace/theme-${this.aceTheme()}.js`).then(() => {
```

Comment:
We didn't have to load the theme separately before?

Reply:
Yup it was loaded on demand previously which resulted in the "flash". You can see it clearly in the before screenshot I posted. The solution stated in https://github.com/ajaxorg/ace/issues/3286 is to load the theme file first.